base/inference.jl

```diff
@@ -1617,6 +1702,12 @@ function abstract_call(f::ANY, fargs::Union{Tuple{},Vector{Any}}, argtypes::Vect
 
     if istopfunction(tm, f, :promote_type) || istopfunction(tm, f, :typejoin)
         return Type
+    elseif length(argtypes) == 2 && istopfunction(tm, f, :typename)
+        t = argtypes[2]
+        if isa(t, Const) || isType(t)
+            return typename_static(t)
+         end
```

Comment:
So it is. Sorry, I'm used to sublime doing this all automatically for me, but I've been trying to use Atom more. Will fix